git-cms-addpkg

```diff
@@ -70,23 +74,40 @@ while [ "$#" != 0 ]; do
       INITOPTIONS="$INITOPTIONS $1"
       shift;;
     -*)
-      echo Unknown option $1 ; usage 1 ;;
+      $ECHO "git cms-addpkg: unknown option $1"; $ECHO; usage 1;;
     *)
       if [ "$INPUT_FILE" == "" ]; then
         # check out a list of packages
         PACKAGES="$PACKAGES $1"
         shift
       else
         # check out a list of packages via -f / --file FILE
-        echo "git cms-addpkg: you cannot specify a package and input from file at the same time."
-        exit 1
+        $ECHO "git cms-addpkg: you cannot specify a package and input from file at the same time."
+        $ECHO
+        usage 1
       fi
     ;;
   esac
 done
-
 if [ "$PACKAGES" == "" ] && [ "$INPUT_FILE" == "" ] ; then
-  echo "You need to specify at least one package or input file." ; exit 1
+  $ECHO "git cms-addpkg: you need to specify at least one package or input file."
+  $ECHO
+  usage 1
+fi
+
+if [ $VERBOSE == 0 ]; then
+  # send vrbose messages to /dev/null
+  exec {verbose}> /dev/null
```

Comment:
Going through the history of the bash documentation, looks like this was introduced (or at least documented) in bash 4.1